packages/barcode-scanning/ios/Plugin/BarcodeScannerHelper.swift

```diff
@@ -81,10 +81,37 @@ public class BarcodeScannerHelper {
             result["bytes"] = convertDataToJsonArray(rawData)
         }
         result["cornerPoints"] = cornerPointsResult
+        if barcode.calendarEvent != nil {
+            result["calendarEvent"] = extractCalendarEventProperties(barcode.calendarEvent!)
+        }
```

Comment:
```suggestion
        if let calendarEvent = barcode.calendarEvent {
            result["calendarEvent"] = extractCalendarEventProperties(barcode.calendarEvent)
        }
```

This would be a cleaner alternative in Swift to avoid the force unwrap.